src/main/resources/config/liquibase/master.xml

```diff
@@ -17,6 +17,7 @@
     <include file="classpath:config/liquibase/changelog/20240617001400_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240612123000_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240614140000_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20240624091200_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
The entries should be in a chronological order

Reply:
aren't they? am I missing something?

Reply:
ah it didn't show the line below, fixed it :+1;